python/pip.bzl

```diff
@@ -188,6 +189,17 @@ def requirement_{key}(r):
     return "@{name}//:{key}__%s" % _sanitize(r)
 """.format(name=ctx.attr.name, key=k) for k in ctx.attr.values.keys()])
 
+    conditions_str = "    ".join(["""\
+conditions["@pip_deps//:{key}"] = [requirement_repo_{key}(name) + label] if name in requirements_map_{key} else []
+""".format(key=k) for k in ctx.attr.values.keys()])
+    specific_macros += """\
+def requirement_repo(name, label=""):
```

Comment:
My idea for pip_version_proxy is to provide the same interface as pip_import, so that you can change between
```
load("@pip_deps2//:requirements.bzl", "requirement_repo") # pip_import rule
```
and
```
load("@pip_deps//:requirements.bzl", "requirement_repo") # pip_version_proxy rule
```
without needing any additional code changes.

With your change, the interface is not the same any more: pip_import has `def requirement_repo(name)` but pip_version_proxy has `def requirement_repo(name, label="")`.

I think the ideal interface would be to remove `requirement_repo` altogether, and have something like this in both pip_import and pip_version_proxy rules:
```
def requirement(name, binary = None, label = None):
    key = name.lower()
    if key not in _requirements:
        fail("Could not find pip-provided dependency: '%s'" % name)
    if binary:
        return _requirements[key].split(":")[0] + ":entrypoint_" + binary
    elif label:
        return _requirements[key].split(":")[0] + label
    return _requirements[key]
```

Reply:
Ok I think I have it done. Pushing in a moment